purity_fb/purity_fb_1dot11/apis/hardware_api.py

```diff
@@ -3,7 +3,7 @@
 """
     Pure Storage FlashBlade REST 1.11 Python SDK
 
-    Pure Storage FlashBlade REST 1.11 Python SDK, developed by [Pure Storage, Inc](http://www.purestorage.com/). Documentations can be found at [purity-fb.readthedocs.io](http://purity-fb.readthedocs.io/).
```

Comment:
Ah...This is why this PR is roughly twice as large as it should be...We must've missed updating the description for 1.11.